mods/ts/rules/gdi-infantry.yaml

```diff
@@ -87,8 +90,23 @@ JUMPJET:
 	-Crushable:
 	AttackFrontal:
 		Voice: Attack
-	WithInfantryBody:
+	WithInfantryBody@Ground:
```

Comment:
`WithInfantryBody` is inherited from the default, so either remove the `@Ground` so the inherited one is overridden, or add `-WithInfantryBody` above to disable it.